docs/2.0/integration/testng.adoc

```diff
@@ -1,29 +1,41 @@
-= TestNG
+=== TestNG
```

Comment:
@eroshenkoam do you mean reverting to a single `=`?.. if yes, for all the titles or only for this one?